sample-policy.rego

```diff
@@ -0,0 +1,9 @@
+package dataapi.authz
+
+rule[{"action": {"name":"RedactAction", "columns": column_names}, "policy": description}] {
```

Comment:
I don't think this file should be committed.

Reply:
Removed